Adyen/Model/Checkout/PayPalDetails.cs

```diff
@@ -0,0 +1,84 @@
+﻿#region Licence
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+#endregion
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Adyen.Model.Checkout
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [DataContract]
+    public class PayPalDetails : IOneOfPaymentRequestPaymentMethod
```

Comment:
IOneOfPaymentRequestPaymentMethod let's remove this name to IPaymentMethodDetails, let do this for all payment methods.